lib/src/material_tree/material_tree_filter.dart

```diff
@@ -75,6 +76,10 @@ class MaterialTreeFilterComponent {
     materialInput.focus();
   }
 
+  void clear() {
+    inputText = '';
+  }
```

Comment:
IIRC, as of 1.24, this can be written more compactly as
```dart
  void clear() => inputText = '';
```

Reply:
We'll probably leave this. Dart style doesn't force one or the other, and individuals have different thoughts on the syntax. Also not worth another sync/CL submission for it.